lib/ansible/modules/cloud/amazon/ecs_taskdefinition_facts.py

```diff
@@ -0,0 +1,346 @@
+            type: complex
+            contains:
+                name:
+                    description: The type of the ulimit .
+                    returned: when present
+                    type: string
+                softLimit:
+                    description: The soft limit for the ulimit type.
+                    returned: when present
+                    type: int
+                hardLimit:
+                    description: The hard limit for the ulimit type.
+                    returned: when present
+                    type: int
+        logConfiguration:
+            description: The log configuration specification for the container.
+            returned: when present
+            type: string
+        options:
+            description: The configuration options to send to the log driver.
+            returned: when present
+            type: string
+family:
+    description: The family of your task definition, used as the definition name
+    returned: always
+    type: string
+taskDefinitionArn:
+    description: ARN of the task definition
+    returned: always
+    type: string
+taskRoleArn:
+    description: The ARN of the IAM role that containers in this task can assume
+    returned: when role is set
+    type: string
+networkMode:
+    description: Network mode for the containers
+    returned: always
+    type: string
+revision:
+    description: Revision number that was queried
+    returned: always
+    type: int
+volumes:
+    description: The list of volumes in a task
+    returned: always
+    type: complex
+    contains:
+        name:
+            description: The name of the volume.
+            returned: when present
+            type: string
+        host:
+            description: The contents of the host parameter determine whether your data volume persists on the host container instance and where it is stored.
+            returned: when present
+            type: bool
+        sourcePath:
+            description: The path on the host container instance that is presented to the container.
+            returned: when present
+            type: string
+status:
+    description: The status of the task definition
+    returned: always
+    type: string
+requiresAttributes:
+    description: The container instance attributes required by your task
+    returned: when present
+    type: complex
+    contains:
+        name:
+            description: The name of the attribute.
+            returned: when present
+            type: string
+        value:
+            description: The value of the attribute.
+            returned: when present
+            type: string
+        targetType:
+            description: The type of the target with which to attach the attribute.
+            returned: when present
+            type: string
+        targetId:
+            description: The ID of the target.
+            returned: when present
+            type: string
+placementConstraints:
```

Comment:
We typically convert return results to snake case. ECS must be weird, it returns dromedaryCase (most things return full CamelCase). Still, I see no reason why camel_dict_to_snake_dict shouldn't work

Reply:
Alright, should be fixed now.